updateList.php

```diff
@@ -0,0 +1,39 @@
+<?php
+session_start();
+
+if(isset($_GET['id'])){
+    $hobbieName = $_GET['id'];
+
+if($hobbieName === "Dancing"){$hobbieID = 1;}
+if($hobbieName === "Listening Music"){$hobbieID = 2;}
+if($hobbieName === "Cricket"){$hobbieID = 3;}
+if($hobbieName === "Singing"){$hobbieID = 4;}
+if($hobbieName === "Fooseball"){$hobbieID = 5;}
+if($hobbieName === "Reading Novels"){$hobbieID = 6;}
+
+$selfID = $_SESSION['id'];
+
+$conn = mysqli_connect("localhost", "root", "abiit@2019", "rconnect");
```

Comment:
shift this connection above so you don't have to write it inside every if condition

Reply:
Done!